.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Intrinsic Support
+    url: https://github.com/rust-lang/stdarch/issues
+    about: Please direct issues about Rust's support for intrinsics to core::arch
+  - name: Internal Compiler Error
+    url: https://github.com/rust-lang/rust/issues
+    about: Please report ICEs to the rustc repository
```

Comment:
ICEs may also be relevant here?  If anyone is going to fix a stdsimd related ICE, it's probably us.

Reply:
Since the compiler CLI already points ICEs back to the `rust-lang/rust` repo maybe we should keep that and help out over there as they come up?

Reply:
Yeah, ICEs are pretty much by definition issues with the compiler. We should obviously fix anything that primarily is an issue for SIMD-related code, though.